docs/Building_with_easystack.rst

```diff
@@ -0,0 +1,143 @@
+.. building_with_easystack:
+
+Building with Easystack
+=======================
+
+This documentation covers aspects of using Easybuild with *Easystack*.
+
+*Note: this is an experimental feature. Some of the mentioned functionality may be subject to change or be prone to errors.*
+
+.. contents::
+    :depth: 3
+    :backlinks: none
+
+
+.. building_with_easystack_basics:
+
+The Basics
+----------
+
+Building with *Easystack* lets user create a file that specifies all the software they want to build, and call EasyBuild with that file as argument.
+EasyBuild then loops over the software specified in the spec file to build it. 
+
+
+.. building_with_easystack_usage:
+
+Usage
+-------------------------
+
+To build software with *Easystack*, type:
+
+.. code::
+
+  eb --easystack yourEasystackFile.yaml
+
+where yourEasystackFile.yaml is the file with specifications that you just created (more about this in the next chapter).
+
+.. building_with_easystack_structure:
+
+Structure of an Easystack file
+-------------------------
+
+Easystack file can currently be of type YAML. 
+
+General options, which should be applied to each software (i.e. *robot*), are defined at the top of the file.
+
+Afterwards, particular software specifications follow.
+
+It is mandatory to specify basic software-related keywords - *software name*, *toolchain* and *versions*.
+
+*software name*: Name of software.
+
+*toolchain*: A group of development related utilities (eg compiler) and libraries. 
+(see :ref:`toolchains` for more details).
+
+*versions*: Version of software. If multiple entries are provided, EB will install all of them. 
+Can be in form of a list or consecutive line entries (see example). 
+
+**General structure of YAML-formatted *easystack*:**
+
+.. code::
+
+  software:
+    software_name:
+      toolchain:
+        toolchain_name (incl. version):
+            software_version:
+              versionsuffix:
+
+**Use-case example of YAML-formatted *easystack*:**
+
+.. code::
+
+  software:
+    Bioconductor:
+      toolchains:
+        foss-2020a:
+          versions:
+            3.11:
+    EasyBuild:
+      toolchains:
+        SYSTEM:
+          versions: [4.3.1]
+    GROMACS:
+      toolchains:
+        foss-2020a:
+          versions:
+            2020.1:
+            2020.3:
+        fosscuda-2020a:
+          versions: [2020.1]
+    OpenFOAM:
+      toolchains:
+        foss-2020a:
+          versions: [8, v2006]
+    R:
+      toolchains:
+        foss-2020a:
+          versions: [4.0.0]
+
+Installation using this *easystack* called 'myeasystack.yaml' with command
+
+.. code::
+
+  eb --easystack myeasystack.yaml;
+
+is an equivalent of running:
+
+.. code::
+
+  eb Bioconductor-3.11-foss-2020a.eb;
+  eb EasyBuild-4.3.1-SYSTEM.eb;
+  eb GROMACS-2020.1-foss-2020a.eb;
+  eb GROMACS-2020.3-foss-2020a.eb;
+  eb GROMACS-2020.1-fosscuda-2020a.eb;
+  eb OpenFOAM-8-foss-2020a.eb;
+  eb OpenFOAM-v2006-foss-2020a.eb;
+  eb R-4.0.0-foss-2020a.eb;
```

Comment:
I believe it is possible to run all the easyconfigs in one eb command, which is what I think easystack actually currenlty does?
```eb Bioconductor-3.11-foss-2020a.eb EasyBuild-4.3.1-SYSTEM.eb GROMACS-2020.1-foss-2020a.eb  GROMACS-2020.3-foss-2020a.eb GROMACS-2020.1-fosscuda-2020a.eb OpenFOAM-8-foss-2020a.eb OpenFOAM-v2006-foss-2020a.eb R-4.0.0-foss-2020a.eb
```

Reply:
that is true, changed